Remotion/Web/Development.WebTesting/Utilities/ProcessUtils.cs

```diff
@@ -160,14 +249,27 @@ public static void GracefulProcessShutdown ([NotNull] Process process, TimeSpan
         // https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill#System_Diagnostics_Process_Kill
         // This exception can be swallowed safely, as we are sure that a process is associated and the process has
         // already exited, therefore the method can return at this point.
-        return;
       }
+    }
 
-      // No exception is thrown when calling .WaitForExit(..) on an already exited process.
-      if (process.WaitForExit (timeoutInMilliseconds))
-        return;
+    private static void WaitForProcessesExits (List<Process> processes, TimeSpan timeout)
```

Comment:
I think the name of the method should indicate that the collection is modified and exited processes are removed, this aspect is not deducible from the signature, or it could just return a new collection with non-exited processes instead of mutating.